packages/block-library/src/index.js

```diff
@@ -62,9 +62,29 @@ import * as tagCloud from './tag-cloud';
 
 import * as classic from './classic';
 
+/**
+ * Function to register an individual block.
+ *
+ * @param {Object} block    The block to be registered.
+ *
+ */
+const registerBlock = ( block ) => {
+	if ( ! block ) {
+		return;
+	}
+	const { metadata, settings, name } = block;
+	if ( metadata ) {
+		unstable__bootstrapServerSideBlockDefinitions( { [ name ]: metadata } ); // eslint-disable-line camelcase
+	}
+	registerBlockType( name, settings );
+};
+
 /**
  * Function to register core blocks provided by the block editor.
  *
+ * @param {boolean} enableLegacyWidgets     Optional. Editor setting to enable Legacy Widget Block.
+ * @param {boolean} enableMenu              Optional. Editor setting to enable Menu Block.
```

Comment:
Looks like this was refactored and these two params don't seem to be needed.

Reply:
yes, this should be removed.